stdlib/public/SDK/Foundation/NSIndexSet.swift

```diff
@@ -12,7 +12,7 @@
 
 @_exported import Foundation // Clang module
 
-// FIXME: move inside NSIndexSet when the compiler supports this.
+// TODO: Evaluate deprecating with a message in favor of IndexSet.
```

Comment:
@phausler and I couldn't see much of a benefit to moving this into `NSIndexSet` itself — it's unlikely anyone is referring to this type by name, but if they are, the risk of breakage is kind of silly. In reality, you should probably be using `IndexSet` anyway.